src/FSharpPlus/List.fs

```diff
@@ -27,7 +27,26 @@ module List =
         loopM xs
 
     let inline replicateM count (initial: '``Applicative<'T>``)  = sequence (List.replicate count initial)
-
+    /// <summary>This function finds the maxima of a list of values, preserving the sequence of retained values from the supplied list.</summary>
```

Comment:
This looks fine, but it should be in Extensions.fs instead.
This is my fault, again because lack of design guidelines, but just move it there. Non-generic functions are mostly there.

Reply:
Adding a comment about it here could help. What's the purpose of these modules?